packages/expo-modules-core/ios/ReactDelegates/ModulePriorities.swift

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+internal struct ModulePriorities {
```

Comment:
Add a comment with what this is for. We should have well-documented native code too, even for internal uses. The new core must be of good quality 😉 It'll decrease technical debt in the future and make it easier for others to contribute.